contracts/Strategy.sol

```diff
@@ -769,7 +786,9 @@ contract Strategy is BaseStrategyInitializable, ICallee {
     // 0 = no cooldown or past withdraw period
     // 1 = claim period
     // 2 = cooldown initiated, future claim period
-    function _checkCooldown() internal view returns (uint8) {
+    enum CooldownStatus {None, Claim, Initiated}
```

Comment:
minor 

it's usually better to put all enums at the beginning (where you declare state variables)